tests/integration/targets/eos_vrf_global/tests/common/parsed.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- ansible.builtin.debug:
+    msg: START eos_vrf_global parsed integration tests on connection={{ ansible_connection }}
+
+- name: Parse externally provided VRF configuration
```

Comment:
we need to assert the parsed output right ?

Reply:
yep, added!